readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information
+Build: {{ build.id }}
+Project: {{ build.project }}
+Version: {{ build.commit }}
+Date: {{ build.date }}
+State: {{ build.state }}
+{% if build.state == 'finished' %}Success: {{ build.success }}{% endif %}
```

Comment:
Not sure if we should change the format of the file based on state of the build. It feels cleaner to just say "Unknown" here or something, instead of change the format of the file.